### DISCUSSION TITLE
CHANGELOG for humans

### DISCUSSION THREAD

**Discussion by vankhoawin** [twilio-labs/paste#1191] [Q&A] [answered]
in regards to updating Paste dependencies, was trying to look for breaking changes in the CHANGELOG, but in 5.0.0 and 6.0.0 i don't see any breaking changes, just patches. i'm not sure how other open-source libraries do it (hopefully not manually), but as a consumer I should be able to easily scan a CHANGELOG for things to watch out for in major version bumps. 

https://github.com/twilio-labs/paste/blob/main/packages/paste-core/core-bundle/CHANGELOG.md#600

Open-source example from Babel: https://github.com/babel/babel/blob/main/CHANGELOG.md#v700-rc4-2018-08-27

**Comment by SiTaggart** [comment, marked as answer]
Hi @vnguyen94 

We recently switched to Changesets to manage our versioning, but they had a default stance that peer dependency updates invoke a major bump in the consumer. 

We've just enabled a flag in the configuration to stop that from happening in the future, because major version bumps just for peer dependency patch updates is _super_ annoying.

You shouldn't see anymore going forward

**Reply by vankhoawin** [reply]
oh that's _super_ weird default behavior. breaking changes are API changes for the consumer, not peer deps i would think 🤔 thanks for the quick response though, glad it's been resolved

**Reply by SiTaggart** [reply]
Yeah, it's pretty wild. You can find more reading on it here if you're interested:

https://github.com/atlassian/changesets/blob/master/docs/decisions.md#the-versioning-of-peer-dependencies

https://github.com/atlassian/changesets/issues/524
